product_supplierinfo_for_customer/migrations/12.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def _move_model_in_data(env, ids, old_model, new_model):
+    renames = [
+        ('mail_message', 'model', 'res_id'),
+        ('mail_followers', 'res_model', 'res_id'),
+        ('ir_attachment', 'res_model', 'res_id'),
+        ('mail_activity', 'res_model', 'res_id'),
+        ('ir_model_data', 'model', 'res_id'),
+    ]
+    for rename in renames:
+        openupgrade.logged_query(
+            env.cr, """
+            UPDATE {A}
+            SET {B} = '{C}'
+            WHERE {D} IN {E} AND {F} = '{G}'""".format(
+                A=rename[0],
+                B=rename[1], C=new_model,
+                D=rename[2], E=tuple(ids), F=rename[1], G=old_model,
+            ),
+        )
+
+
+def fill_product_customerinfo(env):
+    # In pre-migration the product_customerinfo table still doesn't exist
+    other_fields = ''
+    if openupgrade.table_exists(env.cr, 'purchase_requisition_line'):
+        other_fields += ' purchase_requisition_line_id,'
+    # you must customize 'other_fields' to add fields from your custom modules
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO product_customerinfo (
```

Comment:
You mustn't insert specific fields, but copy full structure and then move full record. This way, you don't need to worry by any possible extension to the model.

See https://github.com/OCA/contract/blob/a0e118c97570dc1c3cef7df236c714d30ee547b6/contract/migrations/12.0.4.0.0/pre-migration.py#L90-L103 as example for this.